_pages/install/integrations/cds-integration.md

```diff
@@ -0,0 +1,239 @@
+;;HomepageE2EMerchandisingCarousel;Homepage E2E Merchandising Carousel;Homepage E2E Merchandising Carousel;10;00000000-0000-0000-0000-000000000000;HomepageE2EMerchandisingCarousel
+
+# Add the component to the slot
+INSERT_UPDATE ContentSlot;$contentCV[unique=true];uid[unique=true];cmsComponents(uid,$contentCV)
+;;Section2CSlot-Homepage;HomepageE2EMerchandisingCarousel
+
+# Category Page carousel
+# Create the CMS Component
+INSERT_UPDATE MerchandisingCarouselComponent;$contentCV[unique=true];uid[unique=true];name;title;numberToDisplay;strategy;&componentRef
+;;CategoryPageE2EMerchandisingCarousel;Category Page E2E Merchandising Carousel;Category Page E2E Merchandising Carousel;10;00000000-0000-0000-0000-000000000000;CategoryPageE2EMerchandisingCarousel
+
+# Add the component to the slot
+INSERT_UPDATE ContentSlot;$contentCV[unique=true];uid[unique=true];cmsComponents(uid,$contentCV)
+;;BottomHeaderSlot;CategoryPageE2EMerchandisingCarousel
+
+# SLR Page carousel
+# Create the CMS Component
+INSERT_UPDATE MerchandisingCarouselComponent;$contentCV[unique=true];uid[unique=true];name;title;numberToDisplay;strategy;&componentRef
+;;SLRCategoryPageE2EMerchandisingCarousel;SLR Category Page E2E Merchandising Carousel;SLR Category Page E2E Merchandising Carousel;10;00000000-0000-0000-0000-000000000000;SLRCategoryPageE2EMerchandisingCarousel
+
+# Add the component to the slot
+INSERT_UPDATE ContentSlot;$contentCV[unique=true];uid[unique=true];cmsComponents(uid,$contentCV)
+;;SLRCamerasBottomHeaderSlot;SLRCategoryPageE2EMerchandisingCarousel
+```
+
+**Note:** The above ImpEx includes examples of locations where you can create and place the carousel in a slot, but you are not limited to these locations. You can place the CMS component in any slot that allows it.
+
+### Adding a Merchandising Carousel with SmartEdit
+
+If you have followed the [SmartEdit Setup Instructions for Spartacus]({{ site.baseurl }}{% link _pages/install/smartEdit-setup-instructions-for-spartacus.md %}), you can use SmartEdit to add a Context-Driven Merchandising carousel to a page, as follows:
+
+1. Launch SmartEdit.
+
+2. Edit the page you want to add a Merchandising carousel to.
+
+3. Click the **+ Component** button in SmartEdit.
+
+   You should see the **SAP CONTEXT-DRIVEN MERCHANDISING** carousel component. If you do not see it, use the search box.
+
+4. Drag and drop the component onto a content slot on the page.
+
+   If the component does not drop in the **Basic Edit** mode, switch to the **Advanced Edit** mode using the dropdown in SmartEdit.
+
+5. Fill in the fields of the configuration window that appears.
+
+   For the **Number to Display** field, specify the maximum number of products the carousel can retrieve from Merchandising. If the number of products exceeds the size of the carousel, the user will need to use scrolling arrows to see all products.
```

Comment:
Can this be changed to be:

'Specify the number of products the user wishes to show in this carousel'?

My worry otherwise is someone will think, I have 20k products in my catalog, let's try to request it.

Reply:
Haha, good point 😆 
Updated to say "Specify the number of products you wish to display in the carousel."